beast/physicsmodel/prior_weights_stars.py

```diff
@@ -4,6 +4,7 @@
 The priors on age, mass, and metallicty are computed as weights to use
 in the posterior calculations.
 """
+from sys import exit
```

Comment:
Is this actually needed?  I've never had to import exit before to use it in any python code written?  Mainly curious.

Reply:
I added this after running this code:

```
import numpy as np; import matplotlib.pylab as pl
import beast
from beast.physicsmodel.prior_weights_stars import compute_age_prior_weights

x=np.arange(6.0,8.01,0.1)
mod={"name": "bins_hist", "values":[2.0,1.0,2.5,1.5,1.0], "logages":np.arange(6.0,8.01,0.5)}
p_age = compute_age_prior_weights(x,mod)
```
resulting in this output & error:

```
input age prior ''bins_hist'' function not supported
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-14-a84260952a1f> in <module>()
----> 1 p_age = compute_age_prior_weights(x,mod)

~/work/gitproj/beast/beast/physicsmodel/prior_weights_stars.py in compute_age_prior_weights(logages, age_prior_model)
     70             )
     71         )
---> 72         exit()
     73 
     74     # normalize to avoid numerical issues (too small or too large)

NameError: name 'exit' is not defined
```

Can confirm it is needed from this.  The import only appears in a couple files (e.g., beast/tools/verify_params.py) so could use a few more added.  I'll add the list as an easy issue (see #427), and add the import to prior_weights_dust.py as that is in-scope for this PR.

Reply:
Interesting.  No worries, just have never gotten that error when running from the command line.  Wonder if it is due to using ipython.  Fine with adding it.